packages.yml

```diff
@@ -1,3 +1,5 @@
 packages:
   - package: fivetran/hubspot_source
+    version: [">=0.8.0", "<0.9.0"]
+  - package: calogica/dbt_expectations
     version: [">=0.8.0", "<0.9.0"]
```

Comment:
should add this to the list of package dependencies in the README

Reply:
Great catch! Will update.

Reply:
Updated and added dbt-date as well.